src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -114,11 +196,22 @@ public unsafe void Destroy()
 
             private unsafe IntPtr AsRuntimeDefined(in Guid riid)
             {
-                if ((Flags & CreateComInterfaceFlags.CallerDefinedIUnknown) == CreateComInterfaceFlags.None)
+                int i = UserDefinedCount;
+                if ((Flags & CreateComInterfaceFlagsEx.CallerDefinedIUnknown) == 0)
                 {
                     if (riid == IID_IUnknown)
                     {
-                        return (IntPtr)(Dispatches + UserDefinedCount);
+                        return (IntPtr)(Dispatches + i);
+                    }
+
+                    i++;
+                }
+
+                if ((Flags & CreateComInterfaceFlagsEx.TrackerSupport) != 0)
+                {
+                    if (riid == IID_IReferenceTrackerTarget)
+                    {
+                        return (IntPtr)(Dispatches + i);
                     }
```

Comment:
```suggestion
                    }

                    i++;
```
For symmetry, in case more interfaces are added later.